docs/dev/table/functions.md

```diff
@@ -316,7 +316,7 @@ value NOT IN (sub-query)
 </table>
 </div>
 
-<div data-lang="java" markdown="1">
+<div data-lang="Java/Python" markdown="1">
```

Comment:
In line 432 "STRING.similar(STRING)" is previous wrong.Can you change it along the way both in this file and corresponding zh.md file?Thank you.
STRING.similar(STRING) -> STRING1.similar(STRING2)